xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitlesLibass.cpp

```diff
@@ -22,9 +22,15 @@
 
 namespace
 {
+
+// Directory where user defined fonts are located (and where mkv fonts are extracted to)
+const std::string userFontPath = "special://home/media/Fonts/";
```

Comment:
These might be better as `constexpr const char*` but it doesn't make much difference.